spec/acceptance/nodesets/el6_server.yml

```diff
@@ -0,0 +1,48 @@
+<%
+  hypervisor = ENV['BEAKER_HYPERVISOR'] || 'vagrant'
+
+  case ENV['BEAKER_OS']
```

Comment:
Might want to make this `SIMP_BEAKER_OS` or something unless that's an actual Beaker environment variable.